cmake/benchmark.pc.in

```diff
@@ -7,6 +7,5 @@ Name: @PROJECT_NAME@
 Description: Google microbenchmark framework
 Version: @VERSION@
 
-Libs: -L${libdir} -lbenchmark
-Libs.private: -lpthread
```

Comment:
we need to keep this setting to Libs.private though please.  pthread is only required to link benchmark, not to link _against_ benchmark.

Reply:
As far as I can tell (if I understand you correctly), `-lpthread` _is_ required when linking against benchmark. Based off of that conclusion it should be redundant to b in both Libs.private and Libs.

I was only using `pkg-config` inside of a `Makefile` and my system (glibc 2.31) would not link the code that depends on `-lbenchmark` unless I added in `-lpthreads` myself. For example, when I build the demonstration file from the README using ``g++ demo.cpp `pkg-config --cflags --libs benchmark` -o demo`` with benchmarks v1.8.0 I get

```
/usr/bin/ld: /usr/local/lib/libbenchmark.a(benchmark_runner.cc.o): in function `benchmark::internal::BenchmarkRunner::DoNIterations()':
benchmark_runner.cc:(.text+0x1418): undefined reference to `pthread_create'
collect2: error: ld returned 1 exit status
```

Reply:
`libbenchmark.a` is a *static* library, it itself isn't linked to the `-lpthread`,
so naturally, when linking to the static library, you need to also link to all the libraries
said library would have linked against. This isn't the case for shared libraries.

Reply:
Well that might explain why `pkg-config --static --libs` brings out `-lpthread`... Please hold.